workshops/image_editor/README.md

```diff
@@ -0,0 +1,451 @@
+
+![Image editor's layour](https://cloud-lrjvj5rqz.vercel.app/0image.png)
+1. In **section 1** we will put an image input and a button that updates the image displayed in section 2.
+2. In **section 2** the chosen image will be displayed.
+3. In **section 3** we will put some inputs that will let the user edit the image displayed in section 2.
+
+### Coding the layout
+We're going to use [Boostrap](https://getbootstrap.com), a library that makes CSS styling easy, to easily make this layout.
+
+Add the following code inside of your `<head>` tag. This links the HTML file to the Bootstrap CSS file, and it'll give our app a little bit of cool styling without us having to do it ourselves.
+
+```html
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootswatch/4.5.2/cerulean/bootstrap.min.css" integrity="sha384-3fdgwJw17Bi87e1QQ4fsLn4rUFqWw//KU0g8TvV6quvahISRewev6/EocKNuJmEw" crossorigin="anonymous">
+```
+[Bootstrap’s grid system](https://getbootstrap.com/docs/4.1/layout/grid/) uses a series of containers, rows, and columns to layout and align content. Our website will be full of them:
+![Div Containers](https://cloud-e8d17xj1f.vercel.app/0image.png)
+- Color grey represents the ```<body>``` tag
+- Color green represents the main container.
+- Color pink represents a row.
+- Color blue represents a column, the two last rows are divided by 3 columns.
+
+Add the following code inside your `<body>` tag:
+
+```html
+<div class="container-fluid">
+
+    <!--Section #1-->
+  <div class="row">
+    <div class="col-md-12">
+    </div>
+  </div>
+  
+  <!--Section #2-->
+  <div class="row">
+    <div class="col-md-4">
+    </div>
+    <div class="col-md-4">
+    </div>
+    <div class="col-md-4">
+    </div>
+  </div>
+
+    <!--Section #3-->
+  <div class="row">
+    <div class="col-md-4">
+    </div>
+    <div class="col-md-4">
+    </div>
+    <div class="col-md-4">
+    </div>
+  </div>
+  
+</div>
+```
+Each ```<div>``` tag simply creates a container where the rows/columns will be in. What we actually need to understand are the classes that we are using:
+
+- The main container has the class **container**-**fluid**, which is like any other div but that can expand/shrink depending on how big your browser is displaying the website. This is what's known as [responsive web design](https://www.w3schools.com/html/html_responsive.asp).
+- Column classes indicate the number of columns you’d like to use out of the possible 12 per row. We need three equal-width columns across, so we can use  ```col-md-4```. We also need a single column for the first row, which is why we are using ```col-md-12```.
+
+## Section 1
+In this section we are going to put an input where people will be able to choose what image they want to edit. To do this we are going to need a form, a text input and a button with these values:
+```html
+<!--Section #1-->
+<div class="row">
```

Comment:
The indentation here seems to be inconsistent with the rest of the file. Everywhere else you have 2 spaces, but here you have 4 spaces